mirrors.yaml

```diff
@@ -2,705 +2,780 @@
-      - Dallas, Texas, United States
-      - Dongguan, Guangdong, China
-      - Foshan, Guangdong, China
-      - Frankfurt, Hesse, Germany
-      - Guangzhou, Guangdong, China
-      - Hohhot, Inner Mongolia, China
-      - Hong Kong
-      - Huizhou, Guangdong, China
-      - Kiev, Ukraine
-      - Langfang, Hebei, China
-      - London, United Kingdom
-      - Madrid, Comunidad de Madrid, Spain
-      - Melbourne, Victoria, Australia
-      - Metropolitan City of Milan, Lombardy, Italy
-      - Moscow, Russian Federation
-      - Mumbai, Maharashtra, India
-      - Nanjing, Jiangsu, China
-      - Paris, Ile-de-France, France
-      - Qingdao, Shandong, China
-      - San Jose, California, United States
-      - Seattle, Washington, United States
+      - Chongqing, China
+      - Fujian, China
+      - Gansu, China
+      - Guangdong, China
+      - Guangxi, China
+      - Guizhou, China
+      - Hainan, China
+      - Hebei, China
+      - Heilongjiang, China
+      - Henan, China
+      - Hubei, China
+      - Hunan, China
+      - Inner Mongolia, China
+      - Jiangsu, China
+      - Jiangxi, China
+      - Jilin, China
+      - Liaoning, China
+      - Ningxia, China
+      - Qinghai, China
+      - Shaanxi, China
+      - Shandong, China
       - Shanghai, China
-      - Shenzhen, Guangdong, China
-      - Shijiazhuang, Hebei, China
-      - Singapore
-      - Sterling, Virginia, United States
-      - Stockholm, Stockholm County, Sweden
-      - Suzhou, Jiangsu, China
-      - Sydney, New South Wales, Australia
+      - Shanxi, China
+      - Sichuan, China
       - Tianjin, China
+      - Tibet, China
+      - Xinjiang, China
+      - Yunnan, China
+      - Zhejiang, China
+      - France
+      - Germany
+      - Hong Kong
+      - Bangalore, India
+      - Chennai, India
+      - Hyderabad, India
+      - Kolkata, India
+      - Mumbai, India
+      - Nagpur, India
+      - New Delhi, India
+      - Patna, India
+      - Batam, Indonesia
+      - Jakarta, Indonesia
+      - Surabaya, Indonesia
+      - Italy
+      - Osaka, Japan
       - Tokyo, Japan
-      - Wuhan, Hubei, China
-      - Wuxi, Jiangsu, China
-      - Yancheng, Jiangsu, China
-      - Yangzhou, Jiangsu, China
-      - Zhongshan, Guangdong, China
+      - Seoul, South Korea
+      - Kuwait
+      - Kuala Lumpur, Malaysia
+      - Netherlands
+      - Nigeria
+      - Oman
+      - Pakistan
+      - Manila, Philippines
+      - Doha, Qatar
+      - Singapore
+      - Johannesburg, South Africa
+      - Spain
+      - Sweden
+      - Bangkok, Thailand
+      - Turkey
+      - Ukraine
+      - United Arab Emirates
+      - United Kingdom
+      - Ashburn, United States
+      - Chicago, United States
+      - Dallas, United States
+      - Denver, United States
+      - Los Angeles, United States
+      - Miami, United States
+      - New York, United States
+      - San Jose, United States
+      - Santa Clara, United States
+      - Seattle, United States
```

Comment:
Please refer to the commit message. I only changed the way of writing place names.